File: API/Features/Expenses/Ledgers/ViewModels/Report/LedgerVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Expenses.Ledgers {

    public class LedgerVM {

        public string Date { get; set; }
        public SimpleEntity Supplier { get; set; }
        public DocumentTypeVM DocumentType { get; set; }
        public PaymentMethodVM PaymentMethod { get; set; }
        public bool HasDocument { get; set; }
        public string DocumentName { get; set; }
        public string InvoiceNo { get; set; }
        public decimal Amount { get; set; }
        public decimal Debit { get; set; }
        public decimal Credit { get; set; }
        public decimal Balance { get; set; }

    }

}

File: FrontEnd/src/app/features/expenses/ledgers/classes/view-models/list/ledger-vm.ts
import { SimpleEntity } from 'src/app/shared/classes/simple-entity'

export interface LedgerVM {

    id: string
    date: string
    formattedDate: string
    supplier: SimpleEntity
    documentType: SimpleEntity
    hasDocument: boolean
    documentName: boolean
    invoiceNo: string
    debit: number
    credit: number
    balance: number

}
